pages.es/common/simplehttpserver.md

```diff
@@ -0,0 +1,25 @@
+# simplehttpserver
+
+> Un simple servidor HTTP/S que soporta subida de ficheros, autenticación básica, y reglas YAML para respuestas personalizadas.
+> Una alternativa Go a `http.server` de Python.
+> Más información: <https://github.com/projectdiscovery/simplehttpserver>.
+
+- Inicia el servidor HTTP que sirve el directorio actual con salida verbose (escucha en todas las interfaces y puerto 8000 por defecto):
```

Comment:
```suggestion
- Inicia el servidor HTTP que sirve el directorio actual y muestra información de las solicitudes (escucha en todas las interfaces y puerto 8000 por defecto):
```

The word `verbose` does not exist and, apparently, [the option toggles output with more information](https://github.com/projectdiscovery/simplehttpserver/blob/main/pkg/httpserver/loglayer.go) (e.g. remote address).

What do you think @vitorhcl @kant?

Reply:
"verbose" is an English adjective that means "using or containing more words than are necessary", according to the [Cambridge Dictionary definition](https://dictionary.cambridge.org/dictionary/english/verbose). In the tech context, it has gained another meaning: "The term is often used as the name of a function that reports more information about an operation" ([source](https://www.pcmag.com/encyclopedia/term/verbose))

However, the Spanish word [verboso](https://dle.rae.es/verboso) does exist, although it's not that commonly used.

IMO the term is understandable for people in inserted in the UNIX/*NIX context, but I think I can't speak myself as I'm not a Spanish native speaker and I don't have much contact with their culture.

Reply:
In computing, Verbose mode is an option available in many computer operating systems and programming languages that provides additional details as to what the computer is doing and what drivers and software it is loading during startup or in programming it would produce detailed output for diagnostic purposes thus makes a program easier to debug. [Source](https://en.wikipedia.org/wiki/Verbose_mode)

Reply:
Another [link](https://forum.wordreference.com/threads/its-a-little-wordy.548982/post-16897731) related with verbose